website/versioned_docs/version-0.13/assets/architecture/.gitignore

```diff
@@ -0,0 +1,9 @@
+*.out
```

Comment:
It now seems that every versioned doc has a full copy of the assets folder, which I guess makes sense. However, it seems to be highly redundant, and very difficult to review. Do you have any ideas how we can make all old doc versions refer to the same set of asset files?

Reply:
fixed in 59837595ac0a3b8152da2cd975f5e2e3ebf30b66, the md files had links to images as ```(assets/*.png)```, now I've replaced all the docs and versioned docs to use the ```(/docs/assets/*.png)``` utilising the assets in ```website/static/docs/assets```. I've also removed the now unnecessary assets folders in each versioned doc thus reducing redundancy.